charts/s1-agent/values.yaml

```diff
@@ -94,6 +97,29 @@ agentInjection:
       cpu: 100m
       memory: 800Mi
 
+admissionControllers:
+  validating:
+    selector:
+      namespaceSelector:
+        matchExpressions:
+          - key: kubernetes.io/metadata.name
+            operator: NotIn
+            values:
+              - kube-system
+              - kube-public
```

Comment:
I know I suggested this selector, but if we will need to output a warning, I guess we'll need to remove this from here and only use the filtering in the helper.

Reply:
unlike agent injection where the selector selects only namespaces with injection label, in this case the default is monitoring everything, so to be on the safe side I think we shouldn't count on the helper to make the right decision when it comes to the cluster functionality